recipes-hulks/aliveness/aliveness.bb

```diff
@@ -5,185 +5,188 @@ LIC_FILES_CHKSUM = "file://${WORKDIR}/git/COPYING;md5=5b4473596678d62d9d83096273
 
 inherit cargo
 
-SRC_URI = "git://github.com/hulks/hulk;branch=main;protocol=https"
-SRCREV = "cce0d1af62f8126f6e51e648607d30d0c30a3657"
+SRC_URI = "git://github.com/julianschuler/hulk;branch=hula-dbus;protocol=https"
+SRCREV = "d5dbc8746e9a4d4af5a1412f389f855d93ca575f"
 S = "${WORKDIR}/git/tools/aliveness"
 
-inherit systemd
-
+SYSTEMD_PACKAGES = "${PN}"
 SYSTEMD_SERVICE:${PN} = "aliveness.service"
+SRC_URI += "file://aliveness.service"
+
+inherit systemd
 
 do_install:append () {
   install -d "${D}${systemd_unitdir}/system"
   install -m 0644 "${WORKDIR}/aliveness.service" "${D}${systemd_unitdir}/system/"
 }
 
+FILES:${PN} += "${systemd_unitdir}/system/aliveness.service"
```

Comment:
Isn't the systemd bbclass adding this automatically? Why did that build before?

Reply:
This is the same as in the other bitbake files. I removed it now from all of them.